content/blog/2019/2019-09-28-on-expected-vomiting-time.md

```diff
@@ -0,0 +1,90 @@
++++
+author="Janez Demšar"
+date= '2019-09-28'
+draft= true
+title="On Expected Vomiting Time"
+type="blog"
+thumbImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+frontPageImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+categories=["model performance", "confusion matrix"]
+shortExcerpt = "Computing meaningful performance scores of models should be creative"
+longExcerpt = "A report of an interesting ending of a lecture about setting probability thresholds for predictive models"
++++
+
+We just finished a lecture and a student came with a question. The lecture was, in short, about how predictive models compute probabilities, and to make decisions we must find a threshold that will minimize the cost, maximize the profit, strike a good balance between sensitivity and specificity, or increase the general human well-being in some other way. So this student came with a question about something I didn't explain well, and she gave me one last chance.
+
+I had some data that looked like this.
+
+| ID | predicted probabilities | аctual class |
```

Comment:
The table looks ok to me, but it doesn't get rendered correctly. I cannot figure out why... 🤔